dask_jobqueue/core.py

```diff
@@ -498,6 +512,30 @@ def scale(self, n=None, jobs=0, memory=None, cores=None):
     def adapt(
         self, *args, minimum_jobs: int = None, maximum_jobs: int = None, **kwargs
     ):
+        """ Scale Dask cluster automatically based on scheduler activity.
+
+        Parameters
+        ----------
+        minimum_cores : int
+           Minimum number of cores for the cluster
+        maximum_cores : int
+           Maximum number of cores for the cluster
```

Comment:
I don't think we want the `_cores` in these names. These will be passed through to https://distributed.dask.org/en/latest/api.html#adaptive which just uses `minimum`/`maximum`.

Reply:
👍 Fixed it ( I think :))